### ISSUE TITLE
Error JSON

### ISSUE THREAD

**Issue by Lucaas333** [hrleinonen/netbox-ipam#1]
-------------------------------------------------------
### Hello!

I went to test your Netbox Scan code and ended up getting the following error message, do you know what it could be?

Netbox version: 3.6.0

Python Version: 3.10

![image](https://github.com/hrleinonen/netbox-ipam/assets/121646437/3684fd17-6b03-4d0d-8f55-3af186ec0d2d)

I changed a few lines in the code so I didn't need to enter data such as API, Network and URL, it looked like this:

![image](https://github.com/hrleinonen/netbox-ipam/assets/121646437/3fa2fcf3-23ac-44f6-bcaf-bbc1a5a68173)

Was it because of the changes? Would you help me?

-------------------------------------------------------

**Comment by hrleinonen** [comment]
Hi,

Thank you for testing. I think I need to add more error logic in my code.
If you add  lines:

print(json_data)
exit()

under the json_data = r.json() line, what is the output?

Regards,

Ville

**Comment by Lucaas333** [comment]
Hello, thanks for getting back!

After putting the lines you told me, the answer I got was:

![image](https://github.com/hrleinonen/netbox-ipam/assets/121646437/f86a8ff0-4a4f-4507-a230-3343cc017457)

**Comment by hrleinonen** [comment]
Hi,

I just noticed, that there is missing "import json". Can you add "import json" and test again.

Regards,

Ville

**Comment by Lucaas333** [comment]
Hello,

What was causing the error I gave you previously was a "/" that I placed after my NetBox URL. Now it is returning the following message:

![image](https://github.com/hrleinonen/netbox-ipam/assets/121646437/25ea2a30-bdb3-4bec-beed-5a3af3df222b)

Edit: I was testing the code without IP's registered in NetBox, when I inserted IP's, it returned to me a message with all the IP's I registered. So JSON is working.

I don't know if I understand the concept of the "Scan" code correctly, but in principle it is not creating the IPs that it locates on the network within the NetBox, is that correct?
